dev-docs/lit-release-process.md

```diff
@@ -0,0 +1,136 @@
+# Lit release process
+
+Audience: Lit core contributors.
+
+## Objective
+
+The end result of this document is a release of all unreleased changes on the main branch for all packages in the Lit monorepo.
+
+> **Note**
+> Before releasing, ensure the main branch has been imported and tested within Google.
```

Comment:
This is unusual, so it's worth a couple sentences about why and how I think. This provides confidence that we're not releasing unexpected breaking changes to our open source users, and Google is unusual because we have an unversioned monorepo.

As for how, someone can ping on the general development channel in Discord

Reply:
Great comment! Done!